apps/api/src/app/events/events.module.ts

```diff
@@ -43,7 +42,7 @@ import { LayoutsModule } from '../layouts/layouts.module';
     ...USE_CASES,
     {
       provide: QueueService,
-      useClass: WorkflowQueueProducerService,
+      useClass: QueueService,
```

Comment:
we still have a dependency on the `QueueService` because the `TriggerEvent` usecase is using it, which is also used by the API `TriggerEventToAll`